src/App.js

```diff
@@ -1,5 +1,6 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
@GedionT is this allowed normally?

Reply:
The second line is enough. Remove the first line.

Reply:
Well I think he removed